internal/host/hostcommands/container_image_availability_cmd.go

```diff
@@ -47,16 +47,11 @@ func (cmd *imageAvailabilityCmd) getImages(ctx context.Context, cluster *common.
 	}
 	images = append(images, mcoImage)
 
-	mustGatherImages, err := cmd.versionsHandler.GetMustGatherImages(cluster.OpenshiftVersion, cluster.CPUArchitecture, cluster.PullSecret)
+	mustGatherImage, err := cmd.ocRelease.GetMustGatherImage(cmd.log, *releaseImage.URL, cmd.instructionConfig.ReleaseImageMirror, cluster.PullSecret)
```

Comment:
Should this also be changed in the [install command](https://github.com/openshift/assisted-service/blob/424f34a42f8f9ca3213090fe2dcdffa0f1d875a2/internal/host/hostcommands/install_cmd.go#L130)?

Also, why do we have two ways to get the must-gather image? Can we get rid of one of them?

Reply:
Talked to @tsorya and the install command can't be changed and he wants this to be as targeted as possible, so this this fine for now.